adapters/repos/db/migrator.go

```diff
@@ -253,6 +253,63 @@ func (m *Migrator) RecalculateVectorDimensions(ctx context.Context) error {
 	return nil
 }
 
+func (m *Migrator) RecountProperties(ctx context.Context) error {
+	count := 0
+	m.logger.
+		WithField("action", "recount").
+		Info("Recounting properties, this may take a while")
+
+	clearedShard := map[string]bool{}
```

Comment:
you can use a set `map[string]struct{}` to save one bit per entry!

Total micro-optimization, but good to know :)

Reply:
:D I feel like bool is more meaningful to the next reader